packages/@lwc/integration-karma/test/template/directive-for-each/index.spec.js

```diff
@@ -75,13 +77,44 @@ it('should render an array of objects with null prototype', () => {
     expect(elm.shadowRoot.querySelector('span').textContent).toBe('text');
 });
 
-it('logs an error when passing an invalid key', () => {
-    const elm = createElement('x-test', { is: XTest });
-    elm.items = [{ key: null, value: 'one' }];
+const scenarios = [
+    {
+        testName: 'dynamic text node',
+        Ctor: XTest,
+        tagName: 'x-test',
+    },
+    {
+        testName: 'static text node',
+        Ctor: XTestStatic,
+        tagName: 'x-test-static',
+    },
+    {
+        testName: 'custom element',
+        Ctor: XTestCustomElement,
+        tagName: 'x-test-custom-element',
+    },
+];
```

Comment:
Testing the three scenarios we care about: Elements, CustomElements, and VStatics.